src/budges/budges.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './budges.css'
+
+const Budge = ({ type }) => {
+    const className = `budge ${type}`
+    return (
+        <div>
+            <button className={className}><span>{type}</span></button>
```

Comment:
badges are usually not interactive elements. I recommend to use `span` element instead